jquery.dataTables/jquery.dataTables-tests.ts

```diff
@@ -261,7 +262,7 @@ $(document).ready(function () {
         {
             ajax: {
                 data: ajaxDataFunc,
-                dataSrc: function (data) { },
+                dataSrc: function (data: any) { },
```

Comment:
I added `any` there and a few other places in test file to resolve implicit `any` since the definition and test files are compiled with `--noImplicitAny`. http://definitelytyped.org/guides/contributing.html